### ISSUE TITLE
full model fineturn

### ISSUE THREAD

**Issue by smith-fugui** [eisneim/ltx_lora_training_i2v_t2v#5]
if i want fineturn whole model how can i do

**Comment by eisneim** [comment]
easy, look at the **ltx_train.py** the most important part is training loop how loss is computed, if you want to do full training you just remove the lora injection and change the trainable parameters to original transformers parameters; 

but, i suggest you do lora first, because you need to have 1 million videos to do full training, if not ,let's say you have 10k video samples, you will destroy the original latent distribution if you don't have enough & diverse dataset

**Comment by CoMo6** [comment]
can you provide your training data, thanks!

**Comment by eisneim** [comment]
@FlyingCan my bullet time training data is self collected 13k video clips  with each around 12 seconds; 

if you just want to train your own LORA, it's very easy to build your own dataset, just one folder of video files, use "precompute.py" to generate; you need to install Ollama first for caption generation

if you want my bullet time video clips for commercial use, pls send me email at spdpd@qq.com
